src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM_NUMBER = new Random();
```

Comment:
This is not a number actually. It's special service that provides random numbers, naming should represent it
```suggestion
    private final Random randomProvider = new Random();
```
or
```suggestion
    private final Random random = new Random();
```
or
```suggestion
    private final Random randomSupplier = new Random();
```
or anything like that